lib/pbench/server/database/alembic/env.py

```diff
@@ -65,6 +31,7 @@ def run_migrations_offline(url: str):
 
     Calls to context.execute() here emit the given string to the script output.
     """
+    url = config.get_main_option("sqlalchemy.url")
```

Comment:
This line seems like it might belong after the comment at lines 20-22.  I like it here, but...if we're going to leave it here, perhaps we should delete that comment.

Reply:
So I think this is two different things.  The `url` is only needed for the offline migration, since `engine_from_config()` takes care of the online setup.

But the comment at line 20 is from the auto-generated code documenting how to get additional configuration parameters if needed.

Reply:
So, based on what I see in #3180 (that posting has been quite useful for understanding things here, thanks!), line 34 here is not really new, and nor is the comment at line 20, so there is no action to be taken here.